packages/gravity/kube-testing/src/spec/transport.ts

```diff
@@ -69,7 +69,8 @@ export class TransportTestPlan implements TestPlan<TransportTestSpec, TransportA
 
     const networkManagerBuilder = new NetworkManagerTestBuilder({
       signalHosts: [{ server: signalUrl }],
-      bridge: spec.transport === 'webrtc-proxy',
+      transport: spec.transport,
+      bridge: spec.transport === 'simplepeer-proxy',
```

Comment:
bridge can be just one of the transports, no need to have a separate options field for it